### ISSUE TITLE
[Windows] Can't download GOG game

### ISSUE THREAD

**Issue by Mylloon** [Heroic-Games-Launcher/HeroicGamesLauncher#1166]
### Describe the bug

When I am trying to download "Serious Sam: The First Encounter" it loads endlessly

### Add logs
```
(21:54:34) INFO:    [Legendary]:       Location: C:\Users\user\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary.exe
(21:54:34) INFO:    [Gog]:             Location: C:\Users\user\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl.exe
(21:54:37) INFO:    [Backend]:         
  Heroic Version: 2.2.6 Oden
  Legendary Version:  0.20.25 Our Benefactors
  OS: Microsoft Windows 11 Professionnel KERNEL: 10.0.22000 ARCH: x64
  CPU: AMD Ryzen 5 3600 6-Core Processor @3.6 
  RAM: Total: 17.1 GB Available: 11.2 GB
  GRAPHICS: GPU0: NVIDIA GeForce RTX 2060 SUPER VRAM: 8192MB DRIVER: 512.15 
  

(21:54:37) INFO:    [Gog]:             Getting data about the user
(21:54:37) WARNING: [Backend]:         Protocol already registered.
(21:54:38) INFO:    [Gog]:             Saved user data to config
(21:54:38) INFO:    [Frontend]:        Refreshing Library
(21:54:38) INFO:    [Gog]:             Getting GOG library
(21:54:38) INFO:    [Gog]:             Number of library pages: 1
(21:54:38) INFO:    [Gog]:             Saved games data
(21:54:38) INFO:    [Legendary]:       Refreshing library...
(21:54:38) INFO:    [Legendary]:       Refreshing Epic Games...
(21:54:38) INFO:    [Legendary]:       Refreshing Epic Games...
(21:54:39) INFO:    [Legendary]:       [cli] INFO: Logging in...
(21:54:39) INFO:    [Legendary]:       [Core] INFO: Trying to re-use existing login session...
(21:54:39) INFO:    [Legendary]:       [cli] INFO: Getting game list... (this may take a while)
(21:54:40) INFO:    [Legendary]:       Updating game list
(21:54:40) INFO:    [Legendary]:       Game List Updated
(21:54:40) ERROR:   [Gog]:             Info command failed "C:\Users\user\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl.exe" info 1207658876 --token="xxx" --lang=en-US --os windows
(21:54:41) ERROR:   [Gog]:             Info command failed "C:\Users\user\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl.exe" info 1207658876 --token="xxx" --lang=en-US --os windows
(21:54:41) INFO:    [Legendary]:       Checking for game updates
(21:54:41) INFO:    [Legendary]:       Found 0 game(s) to update
(21:54:41) INFO:    [Gog]:             Found 0 game(s) to update
(21:54:41) ERROR:   [Gog]:             Info command failed "C:\Users\user\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl.exe" info 1207658876 --token="xxx" --lang=en-US --os windows
(21:54:43) INFO:    [Backend]:         checking for heroic updates
```

### Steps to reproduce

1. Go to Library
2. Try to install "Serious Sam: The First Encounter"
3. See error

### Expected behavior

The game should be installed

### System Information

- OS: Windows 11
- Heroic Version: 2.2.6

**Comment by UnDoxa** [comment]
I'm experiencing the same issue currently. Wanting to download Elder Scrolls IV: Oblivion. +1

I also opened an issue yesterday, but realized that this is the same issue as I have, so its better to comment here for visibility.

[My post below]

Describe the bug

When trying to download a gog game, the download popup does not show up & constantly loads.

Add logs
(12:07:48) INFO: [Legendary]: Location: C:\Users...\AppData\Local\Temp\26EqXQQBmnI6GP1Y7IG3cgNy1ji\resources\app.asar.unpacked\build\bin\win32\legendary.exe
(12:07:48) INFO: [Gog]: Location: C:\Users...\AppData\Local\Temp\26EqXQQBmnI6GP1Y7IG3cgNy1ji\resources\app.asar.unpacked\build\bin\win32\gogdl.exe
(12:07:52) INFO: [Backend]:
Heroic Version: 2.2.6 Oden
Legendary Version: 0.20.25 Our Benefactors
OS: Microsoft Windows 10 Pro KERNEL: 10.0.19044 ARCH: x64
CPU: AMD Ryzen 7 5700U with Radeon Graphics @1.8
RAM: Total: 14.8 GB Available: 10.2 GB
GRAPHICS: GPU0: AMD Radeon(TM) Graphics VRAM: 2048MB DRIVER:

(12:07:52) INFO: [Gog]: Getting data about the user
(12:07:52) WARNING: [Backend]: Protocol already registered.
(12:07:53) INFO: [Gog]: Saved user data to config
(12:07:53) INFO: [Frontend]: Refreshing Library
(12:07:53) INFO: [Gog]: Getting GOG library
(12:07:53) INFO: [Gog]: Number of library pages: 1
(12:07:53) INFO: [Gog]: Saved games data
(12:07:53) INFO: [Legendary]: Refreshing library...
(12:07:53) INFO: [Legendary]: Refreshing Epic Games...
(12:07:53) INFO: [Legendary]: Refreshing Epic Games...
(12:07:55) INFO: [Legendary]: [cli] INFO: Logging in...
(12:07:55) INFO: [Legendary]: [Core] INFO: Trying to re-use existing login session...
(12:07:55) INFO: [Legendary]: [cli] INFO: Getting game list... (this may take a while)
(12:07:55) INFO: [Legendary]: Updating game list
(12:07:55) INFO: [Legendary]: Game List Updated
(12:07:57) INFO: [Legendary]: Checking for game updates
(12:07:57) INFO: [Legendary]: Found 0 game(s) to update
(12:07:57) INFO: [Gog]: Found 0 game(s) to update
(12:07:58) INFO: [Backend]: checking for heroic updates
(12:08:21) ERROR: [Gog]: Info command failed "C:\Users...\AppData\Local\Temp\26EqXQQBmnI6GP1Y7IG3cgNy1ji\resources\app.asar.unpacked\build\bin\win32\gogdl.exe" info 1458058109 --token="eFWanCynLhJdr81q3hAGyh2lD_7zbdbFVChrwRHWGwDKLfSll5qTAPYYGUKiBLKhvMr2eShZfrsAONhy2JGW6OqegNFhf1LZfqEslsRb7MbRc35bSYPPIxgel0dl1KgVza0ABhED31Gi_Hwthw8CMTcYk1Je896voS7eYGjkbp-OsiiyFPIkUguiYq1joNkX" --lang=en-US --os windows
(12:08:52) INFO: [Legendary]: Getting More details with Legendary info command...
(12:08:55) INFO: [Legendary]: Getting Info from Epic API
(12:08:58) ERROR: [Gog]: Info command failed "C:\Users...\AppData\Local\Temp\26EqXQQBmnI6GP1Y7IG3cgNy1ji\resources\app.asar.unpacked\build\bin\win32\gogdl.exe" info 1207658661 --token="eFWanCynLhJdr81q3hAGyh2lD_7zbdbFVChrwRHWGwDKLfSll5qTAPYYGUKiBLKhvMr2eShZfrsAONhy2JGW6OqegNFhf1LZfqEslsRb7MbRc35bSYPPIxgel0dl1KgVza0ABhED31Gi_Hwthw8CMTcYk1Je896voS7eYGjkbp-OsiiyFPIkUguiYq1joNkX" --lang=en-US --os windows
(12:13:12) ERROR: [Gog]: Info command failed "C:\Users...\AppData\Local\Temp\26EqXQQBmnI6GP1Y7IG3cgNy1ji\resources\app.asar.unpacked\build\bin\win32\gogdl.exe" info 1458058109 --token="eFWanCynLhJdr81q3hAGyh2lD_7zbdbFVChrwRHWGwDKLfSll5qTAPYYGUKiBLKhvMr2eShZfrsAONhy2JGW6OqegNFhf1LZfqEslsRb7MbRc35bSYPPIxgel0dl1KgVza0ABhED31Gi_Hwthw8CMTcYk1Je896voS7eYGjkbp-OsiiyFPIkUguiYq1joNkX" --lang=en-US --os windows

Steps to reproduce
Go to 'Library'
Click on 'Gog'
Scroll down to 'Any downloaded GOG game'
Click on 'download'
See continuously loading cycle
Expected behavior
Should show popup of install information.

Screenshots
No response

System Information
OS [e. g. "Window$ 10"]:
Heroic Version [e. g. 2.2.6]:
Additional information
No response

**Comment by MaoDzeDun** [comment]
Just wanted to say, that I had a similar problem on Linux, which ended up me having to logout and log back into my GOG account. I forgot I changed my account password and any attempt to download or open a game in my GOG library would just endlessly load without any indication of the issue. Only after viewing the logs I realized what the problem is. As soon as I logged out and then back in, the issue was resolved. Just leaving this here as a solution for other absentminded people, like myself :).
